### PR TITLE
Update for Ruby Devroom Fosdem2017

### PR DESCRIPTION
**Pull request by toch**

### CODE REVIEW

File: cfp.md [hunk 1]
```diff
@@ -67,9 +67,9 @@ presentation in cases of doubt about your content.
 
 ## Important Dates
 
-* End of the Call for Participation: **December 10th, 2015**
-* Announcement of accepted proposals: **December 17th, 2015**
-* FOSDEM 2016: **January 30th 31st, 2016**
+* End of the Call for Participation: **December 8th, 2016**
+* Announcement of accepted proposals: **December 15th, 2016**
+* FOSDEM 2017: **Frebuary 4th & 5th, 2017**
```
Comment (toch) [inline cfp.md]: you can add that the devroom is on Saturday

File: index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: "FOSDEM &lt;3 Ruby"
 subtitle: "The #1 Ruby language devroom at FOSDEM"
 ---
 
-### [Call for proposals now open!](/cfp)
+### [Call for proposals now open!](/cfp.html)
```
Comment (toch) [inline index.md]: why do you change that?

File: index.md [hunk 3]
```diff
@@ -69,11 +68,11 @@ The Ruby devroom takes place on Sunday, the 31st January 2016, in the room H.221
...
-[Code of Conduct](coc) offers some guidelines to ensure this for both
+[Code of Conduct](coc.html) offers some guidelines to ensure this for both
 attendees and speakers. The organizers are available to anyone who is anything
 less than completely comfortable.
 
 ## How could you support the devroom?
 
 If you'd like to help the Ruby devroom to welcome speakers from all around the
-world, [you do it via sponsorship](/sponsorship.html).
+world, [you do it via sponsorship](sponsorship.html).
```
Comment (toch) [inline index.md]: to be coherent I'd remove the `html` extension

File: index.md [hunk 4]
```diff
@@ -69,11 +68,11 @@ The Ruby devroom takes place on Sunday, the 31st January 2016, in the room H.221
 ## Code of Conduct
 
 We dedicate ourselves to making everyone feel welcome and at home. Our
-[Code of Conduct](coc) offers some guidelines to ensure this for both
+[Code of Conduct](coc.html) offers some guidelines to ensure this for both
```
Comment (toch) [inline index.md]: same here?

File: coc.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ available both during, before and after the conference. You can reach us by
 email, phone, Twitter or simply come to us during the devroom. We'll be easy to
 spot.
 
-* Email: `toch@rubybelgium.be` (Christophe) or `tyler@monkeypox.org` (Tyler)
+* Email: `toch@rubybelgium.be` (Christophe) or `@silvanaprecup` (Silvana)
 * Twitter: [@ruby_fosdem](https://twitter.com/ruby_fosdem)
 * Phone: incoming
 * Emergency number: 112
```
Comment (toch) [inline coc.md]: > we can probably add both the devroom contact & the fosdem contact (which also includes a phone number). What you guys think?

good idea

File: cfp.md [hunk 6]
```diff
@@ -67,9 +67,9 @@ presentation in cases of doubt about your content.
 
 ## Important Dates
 
-* End of the Call for Participation: **December 10th, 2015**
-* Announcement of accepted proposals: **December 17th, 2015**
-* FOSDEM 2016: **January 30th 31st, 2016**
+* End of the Call for Participation: **December 8th, 2016**
+* Announcement of accepted proposals: **December 15th, 2016**
```
Comment (toch) [inline cfp.md]: it will be announced on the 11th by Fosdem so let's sync with it

File: cfp.md [hunk 7]
```diff
@@ -67,9 +67,9 @@ presentation in cases of doubt about your content.
 
 ## Important Dates
 
-* End of the Call for Participation: **December 10th, 2015**
-* Announcement of accepted proposals: **December 17th, 2015**
-* FOSDEM 2016: **January 30th 31st, 2016**
+* End of the Call for Participation: **December 8th, 2016**
```
Comment (toch) [inline cfp.md]: it's very short, we need at least one week to get confirmation that ppl will come or not

I'd do the 4th.

Comment (heliocola) [inline cfp.md]: +1

File: cfp.md (hunk 1 above)
Comment (heliocola) [inline cfp.md]: It is already in index.md

File: index.md (hunk 2 above)
Comment (heliocola) [inline index.md]: somehow without it wasn't working on my local machine so I changed all of them to have the .html

File: coc.md (hunk 5 above)
Comment (heliocola) [inline coc.md]: +1

**Pull request merged**